buildScripts/docs/jsdocx.js

```diff
@@ -376,12 +376,15 @@ jsdocx.parse(options)
 
         Object.keys(structure).forEach(key => {
             path = key.replace(/\./g, '/');
-
-            helper.writeJSON({
-                path  : './docs/output/src/' + path + '.json',
-                indent: 4,
-                force : true
-            }, ns('Neo.' + key));
+            const struct = ns('Neo.' + key);
+
+            if (struct) {
```

Comment:
This is a quick fix to a warning popping when building the docs:
```
(node:21554) UnhandledPromiseRejectionWarning: TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received undefined
    at writeFile (fs.js:1436:5)
    at go$writeFile (./node_modules/graceful-fs/graceful-fs.js:139:14)
    at Object.writeFile (./node_modules/graceful-fs/graceful-fs.js:136:12)
    at ./node_modules/jsdoc-x/node_modules/universalify/index.js:13:12
    at new Promise (<anonymous>)
    at Object.writeFile (./node_modules/jsdoc-x/node_modules/universalify/index.js:7:14)
    at ./node_modules/jsdoc-x/src/lib/helper.js:119:28
```

Reply:
Will create a new ticket for this one.